packages/core/src/common/message-translation.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2017 Ericson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+const Globalize = require("globalize");
+const supportedLocales = ["en", "es", "fr"];
+// const currentLocale = "fr";
+const localeRequire = require("../../i18n/locale.json");
+
```

Comment:
I don't get this ? All it does is require { locale: en } ?

Reply:
This is to mimic the configuration ?

Reply:
Using the locale.json allow us to only read in a file to get the language, then only need t use webpack to instantiate a new language